### PR TITLE
Use prebuilt Docker webservice image

### PR DESCRIPTION
**Pull request by coverbeck**
dockstore/dockstore#3037

Use Docker webservice image from quay.io.

Previously we were building by ADDing
init_webservice.sh and init_migration.sh to the Docker image.
Can't do that on Quay.io, because those files are generated
from templates, so instead pass the files in as volumes.

Note: changed default nonsense value of GITHUB_APP_PRIVATE_KEY_FILE
in compose.config to be an absolute path because a relative path
was causing the test to fail.

For 1.9 -- will not merge for 1.8

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -39,25 +39,22 @@ services:
         awslogs-stream: "elasticsearch"
 
   migration: 
-    build:
-      context: .
-      dockerfile: config/Dockerfile_webservice
+    image: quay.io/dockstore-testing/dockstore-webservice:${DOCKSTORE_VERSION}
```
Comment (denis-yuen) [inline docker-compose.yml]: We should probably get a proper `dockstore` organization for these

File: templates/init_migration.sh.template [hunk 2]
```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+cd "$(dirname "$0")"
```
Comment (denis-yuen) [inline templates/init_migration.sh.template]: could be worth a comment, I don't know without looking it up what this is doing anyway

File: docker-compose.yml (hunk 1 above)
Comment (denis-yuen) [reply docker-compose.yml]: Actually, we have one https://quay.io/organization/dockstore. We should get it moved and a proper gravatar setup @agduncan94

Comment (coverbeck) [reply docker-compose.yml]: Can you please add me to the quay.io dockstore org?

File: templates/init_migration.sh.template (hunk 2 above)
Comment (coverbeck) [reply templates/init_migration.sh.template]: Pushed fix.

File: templates/init_migration.sh.template [hunk 3]
```diff
@@ -1,5 +1,9 @@
 #!/bin/bash
 
+# Make the script's directory the working directory, which is also where
+the webservice JAR is.
```
Comment (denis-yuen) [inline templates/init_migration.sh.template]: think this is splitting onto a new line, making an invalid command

File: templates/init_webservice.sh.template [hunk 4]
```diff
@@ -1,3 +1,7 @@
 #!/bin/bash
 
+# Make the script's directory the working directory, which is also where
+the webservice JAR is.
```
Comment (denis-yuen) [inline templates/init_webservice.sh.template]: think this is splitting onto a new line, making an invalid command

Comment (coverbeck) [reply templates/init_webservice.sh.template]: Yikes, how embarrassing.

Comment (denis-yuen) [reply templates/init_webservice.sh.template]: No worries, looks like the first one is fixed, this one isn't

**Comment by denis-yuen** [comment]
FYI, one of the comments is still broken but approved in principle

**Pull request merged**